src/include/baselib/messaging/BrokerErrorCodes.h

```diff
@@ -75,6 +79,26 @@ namespace bl
                 return false;
             }
 
+            static auto tryGetExpectedErrorMessage( SAA_in const eh::error_code& ec ) NOEXCEPT -> const std::string&
+            {
+                if( ! isExpectedErrorCode( ec ) )
+                {
+                    return bl::str::empty();
+                }
+
+                static const std::string message = ec.message();
```

Comment:
why would you initialize this global message from the first error code (i.e. at random)?

Reply:
Since the suggestion was to return a "std;:string&", I couldn't find another way to return this message.